src/downloadDocumentation.spec.js

```diff
@@ -80,7 +81,59 @@ describe('ServerlessAWSDocumentation', function () {
           },
           accepts: 'application/yaml',
         });
-        expect(objectUnderTest.fs.writeFileSync).toHaveBeenCalledWith('test.yml', 'some body');
+        expect(objectUnderTest.fs.writeFileSync).toHaveBeenCalledWith(outputFileName, 'some body');
+      });
+    });
+
+    it('should successfully download documentation, json extension, using unknown export type', async () => {
+      objectUnderTest.options = {
+        outputFileName: 'test.json',
+        exportType: 'graphql'
+      };
+      objectUnderTest._getRestApiId = () => {
+        return Promise.resolve('testRestApiId')
+      };
+
+      objectUnderTest.serverless.providers.aws.request.mockReturnValue(Promise.resolve({
+        body: 'some body',
+      }));
+      await objectUnderTest.downloadDocumentation().then(() => {
```

Comment:
Not sure I understand why it's necessary to mix up async/await with the promise API... ? It should work without the `.then`

Reply:
You're totally right! I've copy-pasted the structure from [another previously existing test](https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/blob/3379d9144277dbc69ba6f907c49f4083ecf8d1e4/src/downloadDocumentation.spec.js#L48), for both simplicity and consistency with the rest of the code, but somehow I did not notice this.

Fixed in all the tests of this test suite: https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/pull/39/commits/4767337d00d357618c556befad50a9a875b3db1b